README.md

```diff
@@ -1,16 +1,201 @@
+* `report-name`: Optional. If you want to get quick assessments by report, you should create a report before you run the github action and set the report-name value the name of the report you created. At least one of the 2 parameters `report-name` and `deployment-id` must be filled.
+* `deployment-id`: Optional. If you want to get quick assessments by deployment, you should get the id of your deployment, and pass the value to `deployment-id`. At least one of the 2 parameters `report-name` and `deployment-id` must be filled.
+
+
+# End-to-End Sample Workflows
+
+  
+### Sample workflow to apply all  policy file changes in a given directory to Azure Policy
+
+
+```yaml
+# File: .github/workflows/workflow.yml
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Test artifact
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Azure login
+      uses: Azure/login@v1.4.6
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: analyse Microsoft 365 compliance results
+      uses: ./ # Uses an action in the root directory
+      with:
+        cred: ${{ secrets.AZURE_CREDENTIALS }}
+        report-name: 'test-report'
+        
+```
+The above workflow will get assessments by report.
+
+
+### Sample workflow to apply only a subset of assignments from a given directory to Azure Policy
+
+
+```yaml
+# File: .github/workflows/workflow.yml
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Test artifact
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Azure login
+      uses: Azure/login@v1.4.6
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+    - name: Deploy with ARM template
+      id: deployarm
+      uses: azure/arm-deploy@v1
+      with:
+        subscriptionId: ${{ secrets.SUBSCRIPTION_ID }}
+        resourceGroupName: ${{ secrets.RESOURCE_GROUP}}
+        template: ./deploy/Storage.template.json
+        parameters: storageAccountType=Standard_LRS
+    - run: echo ${{ steps.deployarm.outputs.deploymentId }}
+
+    - name: analyse Microsoft 365 compliance results
+      uses: ./ # Uses an action in the root directory
+      with:
+        cred: ${{ secrets.AZURE_CREDENTIALS }}
+        deployment-name: ${{ steps.deployarm.outputs.deploymentId }}
+        
+```
+The above workflow will get quick assessments by deployment. 
+
+
+
+
+
+
+# Configure credentials for Azure login action:
+
+With the Azure login Action, you can perform an Azure login using [Azure service principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals). The credentials of Azure Service Principal can be added as [secrets](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables) in the GitHub repository and then used in the workflow. Follow the below steps to generate credentials and store in github.
```

Comment:
Is it possible directly link to login action's tutorial instead of duplicate one?